sdk/web-pubsub/web-pubsub/src/index.ts

```diff
@@ -7,10 +7,11 @@ export {
   GetAuthenticationTokenResponse,
   HubSendToAllOptions,
   CloseConnectionOptions,
+  GenerateClientTokenOptions,
   HasConnectionOptions,
   HubHasGroupOptions,
   HubHasUserOptions,
-  HubAdminClientOptions,
+  WebPubSubServiceClientOptions as HubAdminClientOptions,
```

Comment:
since we're still in beta, why not rename the public shape to match?

Reply:
VScode F2 is why :-P